tools/packaging/static-build/ovmf/build-ovmf.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+#
+# Copyright (c) 2022 IBM
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+source "${script_dir}/../../scripts/lib.sh"
+
+# disabling set -u because scripts attempt to expand undefined variables
+set +u
+ovmf_repo="${ovmf_repo:-}"
+ovmf_dir="edk2"
+ovmf_version="${ovmf_version:-}"
+kata_version="${kata_version:-}"
+DESTDIR="${DESTDIR:-../../destdir}"
+PREFIX="${PREFIX:-/opt/kata}"
+ovmf_build="${ovmf_build:-sev}"
```

Comment:
The default build should be the "neutral" one, the `x64` one.  Then in the kata-deploy scripts we pass whether we'll build OVMF for SEV or TDVF for TDX.